### PR TITLE
Use same timestamp for multiple images

### PR DESCRIPTION
**Pull request by garyservin**

### CODE REVIEW

**Review by paulbovbel — APPROVED** [review]

File: tailor_image/create_image.py [hunk 1]
```diff
@@ -16,15 +15,18 @@
 
 from . import (
     find_package,
+    merge_dicts,
```
Comment (paulbovbel) [inline tailor_image/create_image.py:18]: Is this not used anywhere?

File: tailor_image/create_image.py [hunk 2]
```diff
@@ -211,7 +214,10 @@ def update_image_index(release_label, apt_repo, common_config, image_name):
         if error.response['Error']['Code'] == 'NoSuchKey':
             click.echo('Index file doesn\'t exist, creating a new one')
 
-    data[date_time] = image_data
+    if timestamp in data:
+        data[timestamp] = merge(data[timestamp], image_data)
```
Comment (paulbovbel) [inline tailor_image/create_image.py:218]: should this be merge_dicts?

Comment (garyservin) [reply tailor_image/create_image.py:218]: yeah, thanks!

File: tailor_image/create_image.py [hunk 3]
```diff
@@ -211,7 +214,10 @@ def update_image_index(release_label, apt_repo, common_config, image_name):
         if error.response['Error']['Code'] == 'NoSuchKey':
             click.echo('Index file doesn\'t exist, creating a new one')
 
-    data[date_time] = image_data
+    if timestamp in data:
```
Comment (paulbovbel) [inline tailor_image/create_image.py:217]: the idiomatic python way here is

```
try:
  ...
except KeyError:
  ...
``

to avoid the redundant 'in' check

**Pull request merged**